packages/f-checkout/src/components/tests/Selector.test.js

```diff
@@ -70,74 +81,73 @@ describe('Selector', () => {
-            propsData
-        });
+                // Assert
+                expect(selector.html()).toMatchSnapshot();
+            });
 
-        const selectorLabel = wrapper.find('[data-test-id="fulfillment-time-label"]');
+            it('should show the collection label when the `serviceType` is `collection`', () => {
+                // Arrange & Act
+                const wrapper = mount(Selector, {
+                    store: createStore({ ...defaultState, serviceType: CHECKOUT_METHOD_COLLECTION }),
+                    i18n,
+                    localVue,
+                    propsData
+                });
 
-        // Assert
-        expect(selectorLabel.html()).toMatchSnapshot();
-    });
+                const selector = wrapper.find('[data-test-id="form-select"]');
 
-    describe('data ::', () => {
-        describe('selectedTime ::', () => {
-            // TODO: Component is working correctly but the tests do not work with `scss modules`
+                // Assert
+                expect(selector.html()).toMatchSnapshot();
+            });
+        });
 
-            xit('should add class to display label above option when not null', async () => {
-                // Arrange
-                const data = { selectedTime: 'testTime' };
+        describe('fulfillmentTimes', () => {
+            it('should create an array of labels from `fulfillment.times` state', () => {
+                // Arrange && Act
                 const wrapper = shallowMount(Selector, {
-                    store: createStore(),
+                    store: createStore({ ...defaultState }),
                     i18n,
                     localVue,
                     propsData
                 });
 
-                const selector = wrapper.find("[data-test-id='form-select']");
-
-                // Act
-                wrapper.setData(data);
-                await wrapper.vm.$nextTick();
+                const expectedTimes = ['time 1'];
 
                 // Assert
-                expect(wrapper.html()).toContain('o-form-select--float');
-                expect(selector.classes()).toContain('o-form-select--float');
+                expect(wrapper.vm.fulfillmentTimes).toEqual(expectedTimes);
             });
+        });
+    });
 
-            it('should remove class to display label centrally when null', () => {
-                // Arrange & Act
+    describe('methods ::', () => {
+        describe('selectionChanged', () => {
+            it('should update `fulfillment.times` time to be selected', () => {
+                // Arrange && Act
                 const wrapper = shallowMount(Selector, {
-                    store: createStore(),
+                    store: createStore({ ...defaultState }),
                     i18n,
                     localVue,
                     propsData
                 });
 
-                const selector = wrapper.find("[data-test-id='form-select']");
+                wrapper.vm.selectionChanged('time 1');
```

Comment:
This is the `// Act`, in this test, no?
```suggestion
                // Act
                wrapper.vm.selectionChanged('time 1');
```